example/lib/main.dart

```diff
@@ -24,16 +22,19 @@ class AudioGame extends BaseGame with TapDetector {
   static Paint gray = PaletteEntry(Color(0xFFCCCCCC)).paint;
   static TextConfig text = TextConfig(color: BasicPalette.white.color);
 
+  late AudioPool pool;
+
   @override
   Future<void> onLoad() async {
-    await pool.init();
+    pool = await AudioPool.create('fire_2.mp3', minPlayers: 3, maxPlayers: 4);
     startBgmMusic();
   }
 
   Rect get button => Rect.fromLTWH(20, size.y - 300, size.x - 40, 200);
 
   void startBgmMusic() {
     FlameAudio.bgm.play('music/bg_music.ogg');
+    FlameAudio.bgm.initialize();
```

Comment:
a bit odd to initialize after playing? probably it doesnt make a difference but feels off